### PR TITLE
9k update

### PR DESCRIPTION
**Pull request by guyboertje**

### CODE REVIEW

File: rakelib/vendor.rake [hunk 1]
```diff
@@ -1,6 +1,7 @@
 namespace "vendor" do
   VERSIONS = {
-    "jruby" => { "version" => "1.7.25", "sha1" => "cd15aef419f97cff274491e53fcfb8b88ec36785" },
+    "jruby" => { "version" => "9.1.6.0", "sha1" => "6dada28876caf2ac14de83dbb45d5243fbbd4940" },
+    # "jruby" => { "version" => "1.7.25", "sha1" => "cd15aef419f97cff274491e53fcfb8b88ec36785" },
```
Comment (suyograo) [inline rakelib/vendor.rake]: Can we remove this all together?

**Comment by suyograo** [comment]
@guyboertje thanks! Can you take a look at the test failures on travis?

File: rakelib/vendor.rake (hunk 1 above)
Comment (guyboertje) [inline rakelib/vendor.rake]: Sure.

**Comment by guyboertje** [comment]
@suyograo 
I have been looking at the Travis fails. Initially I can't decide whether to modify the travis job itself as a fix or to expect that the travis job as is should run and that I must discover that which is preventing travis from running.

**Comment by guyboertje** [comment]
The core specs pass but the integration specs are being hit with this travis "feature" https://github.com/travis-ci/travis-ci/issues/6471

**Comment by ph** [comment]
I will close this and start working on the feature branch instead.

**Pull request closed without merging**